src/config/functions.rs

```diff
@@ -565,12 +565,13 @@ pattern = 'func\d+'
 "#,
         );
         let fns = Functions::parse(Some(&toml), "a");
-        assert_eq!(fns.len(), 4);
```

Comment:
Why did this change?

Reply:
The explanation is in the comment just below. ;)

Reply:
This one: https://github.com/gtk-rs/gir/pull/1027/files#diff-ee42c4f4c96eb712946d909affc0319ee2aa8c12db41446d441a5144191269e4R573 :)

Reply:
Oh the test was broken before...